globals.go

```diff
@@ -57,14 +57,17 @@ const (
 	TaskUnscheduled  = "unscheduled"
 	// TaskWillRun is a subset of TaskUndispatched and is only used in the UI
 	TaskWillRun = "will-run"
-
-	// TaskStarted indicates a task is running on an agent
-	TaskStarted = "started"
+	// TaskContainerAllocated indicates that a tasks's container has been requested
```

Comment:
This will eventually be part of the UI, but the UI for containers/containerized tasks won't exist until [PM-2620](https://jira.mongodb.org/browse/PM-2620). For now, as long as this change is backward compatible with the existing host task logic, it should be fine if it's not accounted for in the UI.